src/Wallabag/CoreBundle/Controller/WallabagRestController.php

```diff
@@ -66,13 +64,15 @@ public function getEntriesAction(Request $request)
         $entries = $this
             ->getDoctrine()
             ->getRepository('WallabagCoreBundle:Entry')
-            ->findEntries($this->getUser()->getId(), $isArchived, $isStarred, $isDeleted, $sort, $order);
+            ->findEntries($this->getUser()->getId(), $isArchived, $isStarred, $sort, $order);
 
         if (!($entries)) {
             throw $this->createNotFoundException();
         }
 
-        return $entries;
+        $json = $this->get('serializer')->serialize($entries, 'json');
+
+        return new Response($json, 200, array('application/json'));
```

Comment:
Well, this is in your doc :)

https://github.com/willdurand/BazingaHateoasBundle/blob/master/Resources/doc/index.md#serializing-objects